### PR TITLE
edits consistent with UI philosophy

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees PTAL

These are edits based on the feedback we got from the UI team in some other recent PRs.

### CODE REVIEW

File: openshift/templates/cakephp.json [hunk 1]
```diff
@@ -230,43 +230,42 @@
...
+      "displayName": "GitHub Webhook Secret",
       "description": "A secret string used to configure the GitHub webhook",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{40}"
     },
     {
       "name": "DATABASE_SERVICE_NAME",
-      "displayName": "Database service name",
-      "description": "Database service name"
+      "displayName": "Database service name"
```
Comment (bparees) [inline openshift/templates/cakephp.json]: some of the display names are initial caps and others are first word caps only.  is that intentional?

Comment (gabemontero) [inline openshift/templates/cakephp.json]: Nope ... I'll address.

**Comment by gabemontero** [comment]
@spadgett FYI another one of these ... I'll apply some of the general comments you had elsewhere in a min

**Comment by gabemontero** [comment]
OK ... applied. PTAL.

File: openshift/templates/cakephp-mysql.json [hunk 2]
```diff
@@ -355,100 +355,96 @@
...
-      "description": "Maximum amount of memory the MySQL container can use",
+      "displayName": "Memory Limit (MySQL)",
+      "description": "Maximum amount of memory the MySQL container can use.",
       "value": "512Mi"
     },
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "displayName": "Source repository URL",
-      "description": "The URL of the repository with your application source code",
+      "displayName": "GIT Repository URL",
```
Comment (spadgett) [inline openshift/templates/cakephp-mysql.json]: "Git Repository URL"

File: openshift/templates/cakephp-mysql.json [hunk 3]
```diff
@@ -355,100 +355,96 @@
...
+      "description": "The URL of the repository with your application source code.",
       "value": "https://github.com/openshift/cakephp-ex.git"
     },
     {
       "name": "SOURCE_REPOSITORY_REF",
-      "displayName": "Source repository reference",
-      "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch"
+      "displayName": "GIT Reference",
```
Comment (spadgett) [inline openshift/templates/cakephp-mysql.json]: "Git Reference"

File: openshift/templates/cakephp.json [hunk 4]
```diff
@@ -230,88 +230,84 @@
...
-      "description": "Maximum amount of memory the container can use",
+      "displayName": "Memory Limit",
+      "description": "Maximum amount of memory the container can use.",
       "value": "512Mi"
     },
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "displayName": "Source repository URL",
-      "description": "The URL of the repository with your application source code",
+      "displayName": "GIT Repository URL",
```
Comment (spadgett) [inline openshift/templates/cakephp.json]: "Git Repository URL"

File: openshift/templates/cakephp.json [hunk 5]
```diff
@@ -230,88 +230,84 @@
...
+      "description": "The URL of the repository with your application source code.",
       "value": "https://github.com/openshift/cakephp-ex.git"
     },
     {
       "name": "SOURCE_REPOSITORY_REF",
-      "displayName": "Source repository reference",
-      "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch"
+      "displayName": "GIT Reference",
```
Comment (spadgett) [inline openshift/templates/cakephp.json]: "Git Reference"

**Comment by gabemontero** [comment]
OK, "GIT" -> "Git" pushed

**Comment by spadgett** [comment]
LGTM

**Comment by bparees** [comment]
lgtm

**Pull request merged**